app/src/controllers/step/actionValidators.js

```diff
@@ -15,7 +15,7 @@
  */
 
 export const validateIgnoreInAA = (item) => {
-  if (item.issue && !item.issue.issueType) {
+  if ((item.issue && !item.issue.issueType) || !item.issue) {
```

Comment:
It can be simplified: `!item.issue || !item.issue.issueType`

Reply:
Fixed